core/tests/webdriverio_desktop/topicAndStoryEditor.js

```diff
@@ -382,3 +385,119 @@ describe('Chapter editor functionality', function() {
+    var handle = await browser.getWindowHandle();
+    await topicsAndSkillsDashboardPage.get();
+    await topicsAndSkillsDashboardPage.createTopic(
+      topicName, topicUrlFragment, 'Description', false);
+    await topicEditorPage.createStory(
+      storyName, 'topicandstoryeditortwo', 'Story description',
+      Constants.TEST_SVG_PATH);
+    await general.closeCurrentTabAndSwitchTo(handle);
+    await users.logout();
+  });
+
+  it('should change the status and publish chapters serially.',
+    async function() {
+      await users.login(userEmail);
+      await topicsAndSkillsDashboardPage.get();
+      await topicsAndSkillsDashboardPage.editTopic(topicName);
+      await topicEditorPage.navigateToStoryWithTitle(storyName);
+      await storyEditorPage.createNewChapter(
+        'Chapter 1', dummyExplorationIds[0], Constants.TEST_SVG_PATH);
+      await storyEditorPage.saveStory('First save');
+      await storyEditorPage.navigateToChapterWithName('Chapter 1');
+      await storyEditorPage.changeNodeDescription('Chapter description 1');
+      await storyEditorPage.setNodePlannedPublicationDate('25-10-2030');
+      await storyEditorPage.changeNodeOutline(
+        await forms.toRichText('First outline'));
+      await storyEditorPage.saveStory('Change chapter details');
+      await storyEditorPage.setNodeStatusToReadyToPublish();
+      await storyEditorPage.navigateToStoryEditorTab();
+      await storyEditorPage.expectChapterStatusToBe(0, 'Ready To Publish');
+
+      await storyEditorPage.createNewChapter(
+        'Chapter 2', dummyExplorationIds[1], Constants.TEST_SVG_PATH);
+      await storyEditorPage.saveStory('Second save');
+      await storyEditorPage.navigateToChapterWithName('Chapter 2');
+      await storyEditorPage.changeNodeDescription('Chapter description 2');
+      await storyEditorPage.changeNodeOutline(
+        await forms.toRichText('Second outline'));
+      await storyEditorPage.setNodePlannedPublicationDate('24-10-2030');
+      await storyEditorPage.saveStory('Change chapter details');
+      await storyEditorPage.setNodeStatusToReadyToPublish();
+      await storyEditorPage.navigateToStoryEditorTab();
+      await storyEditorPage.expectChapterStatusToBe(1, 'Ready To Publish');
+      await storyEditorPage.navigateToChapterWithName('Chapter 2');
+      await storyEditorPage.setNodeStatusToDraft();
+      await storyEditorPage.navigateToStoryEditorTab();
+      await storyEditorPage.expectChapterStatusToBe(1, 'Draft');
+      await storyEditorPage.navigateToChapterWithName('Chapter 2');
+      await storyEditorPage.setNodeStatusToReadyToPublish();
+      await storyEditorPage.navigateToStoryEditorTab();
+      await storyEditorPage.expectChapterStatusToBe(1, 'Ready To Publish');
+
+      await storyEditorPage.publishNodes(1);
+      await storyEditorPage.expectChapterStatusToBe(0, 'Published');
+      await storyEditorPage.expectChapterStatusToBe(1, 'Published');
```

Comment:
@Kej-r03 The test needs some verification. Can you navigate to the learner page and make sure the right things are displayed?

Reply:
Done